bin/picca_deltas.py

```diff
@@ -59,6 +59,13 @@ def cont_fit(data):
     parser.add_argument('--keep-bal',action='store_true',required=False,
         help='Do not reject BALs in drq')
 
+    parser.add_argument('--bal-catalog', type = str, default = None, required=False,
+            help = 'BAL catalog location. Use with --keep-bal to mask BAL features')
+
+    ##New arg for using just BI or AI velocities
+    parser.add_argument('--BAL-index', type = str, default = 'AI', required=False,
```

Comment:
Please use lower case for the options

Reply:
Options in both picca_deltas and py/picca/bal_tools.py should be lowercase now.